recipes/cloudflared/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cloudflared" %}
+{% set version = "2022.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cloudflare/cloudflared/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e49db875da7513d8ae950b20c636225016022866850ff3df2484c0f56cdc4bc4
+
+build:
+  script:
+    - make PREFIX="$PREFIX" VERSION={{ version }} MSI_VERSION= install  # [not win]
+    - make PREFIX=%LIBRARY_PREFIX% VERSION={{ version }} MSI_VERSION= install  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - make
+    - m2-grep  # [win]
+
+test:
+  commands:
+    - cloudflared --help
+
+about:
+  home: https://github.com/cloudflare/cloudflared
+  summary: Cloudflare Tunnel client (formerly Argo Tunnel)
+  description: |
+    Contains the command-line client for Cloudflare Tunnel,
+    a tunneling daemon that proxies traffic from the Cloudflare network to your origins.
+    This daemon sits between Cloudflare network and your origin (e.g. a webserver).
+    Cloudflare attracts client requests and sends them to you via this daemon,
+    without requiring you to poke holes on your firewall --- your origin can remain as closed as possible.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
You need to collect the licenses of the go dependency packages into a file and add this file to the license_file section.

There is a tool which can do collect the licenses for you automatically.

conda-forge/conda-forge.github.io#1052

Reply:
Oops. I forgot that. We should probably also update [the doc](https://conda-forge.org/docs/maintainer/adding_pkgs/#licenses-of-included-dependencies), which currently only include a guide for Rust.

Sorry for the late response!